### PR TITLE
Project Extent | Implement point-drawing in project extent mapping

### PR DESCRIPTION
**Pull request by mddilley**
Closes part of https://github.com/cityofaustin/atd-data-tech/issues/5466

This PR adds the ability to draw points in the Edit Project Extent view accessed through the Project Summary view. With the draw UI, you can add points, select a point, and delete the selected point. The draw UI can be opened by toggling the temporary switch (seen in the capture below). This switch just toggles a piece of state called `isDrawing`. The way this will be toggled in the future is TBD. 

The [next PR](https://github.com/cityofaustin/atd-moped/pull/262) adds the ability to add/remove the drawn points to the project extent stored in the database. To test, click the edit button on the map in https://deploy-preview-257--atd-moped-main.netlify.app/moped/projects/185 and then use the toggle in the bottom left of the edit view.

### Demo
drawPoints

### CODE REVIEW

File: moped-editor/public/static/icon-delete.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
+  <title>icon-delete</title>
+  <path d="M51.4,13.9v1.6c0,0.9-0.7,1.6-1.6,1.6H13.6c-0.9,0-1.6-0.7-1.6-1.6v-1.6c0-0.9,0.7-1.6,1.6-1.6h9
+	c0.9,0,1.6-0.7,1.6-1.6C24.3,9.7,25.1,9,26,9h11.5c0.9,0,1.6,0.7,1.6,1.6c0,0.9,0.7,1.6,1.6,1.6h9C50.7,12.3,51.4,13,51.4,13.9z" />
+  <path d="M40.8,50.1l0.8-25.4h-3.3l-0.8,25.4H40.8z M30.1,50.1h3.3V24.7h-3.3V50.1z M26,50.1l-0.8-25.4h-3.3l0.8,25.4H26
+	z M44.9,55H18.5c-0.9,0-1.6-0.7-1.6-1.6l-1.5-31.2c0-0.9,0.7-1.7,1.6-1.7h29.4c0.9,0,1.7,0.8,1.6,1.7l-1.5,31.2
+	C46.5,54.3,45.8,55,44.9,55z" />
+</svg>
```
Comment (sergiogcx) [inline moped-editor/public/static/icon-delete.svg:8]: 👍

File: moped-editor/src/utils/mapDrawHelpers.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        onSelect={onSelect}
+        clickRadius={12}
+        mode={modeHandler}
+      />
+      {renderDrawToolbar()}
+    </>
+  );
+
+  return { isDrawing, setIsDrawing, renderMapDrawTools };
+}
```
Comment (sergiogcx) [inline moped-editor/src/utils/mapDrawHelpers.js:199]: The whole file is amazing, thanks for the docstrings and showing me how you would approach using custom SVGs. It was very easy to follow the code 👨‍💻 ⚡

**Review by sergiogcx — APPROVED** [review]
Supreme code! 💯 ⚡ 🧑‍💻

**Review by amenity — APPROVED** [review]
LSTM! 🙏✨

**Review by mateoclarke — APPROVED** [review]
🚢

**Pull request merged**